content/foundations/color.mdx

```diff
@@ -2,77 +2,242 @@
+
+As the system grows, it will provide more APIs that fit different color needs. Check the [Primer Primitives](https://github.com/primer/primitives/releases/) repository to follow along as we release new variables and systems.
+
+### Functional system
+
+The functional system is based on the meaning, or purpose, that colors have in the interface. This system is structured into two groups of variables:
+
+- **Foundations:** Foregrounds, backgrounds, and borders that make up most of a product interface. Each variable is generated based on visual weight.
+- **Color roles:** Foregrounds, backgrounds, and borders that highlight affordance or the meaning of elements in the UI.
+
+<img
+  width="960"
+  alt="Accent: blue, Success: green, Attention: yellow, Danger: red, Severe: orange, Done: purple"
+  src="https://user-images.githubusercontent.com/6951037/141302835-05ea2e26-fc89-49ed-8dc5-2af6a6ca9df8.png"
+/>
+
+You should always use Primer's functional color system in your designs to make sure that:
+
+- It supports color modes out of the box
+- It gets all future themes updates "for free"
+- You have an accessible color combination across themes
+
+### Foregrounds
+
+Foreground elements are **text and icons**. You can apply color to them using any of the foreground variables (or `fg` for short).
+
+| Foundations                                                                       | Usage                                                                                                                                |
+| :-------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------- |
+| ![](https://swatch-sid.vercel.app?mode=light&token=fg.muted) `fg.default`         | Primary color for text and icons in any given interface. It should be used for body content, titles and labels.                      |
+| ![](https://swatch-sid.vercel.app?mode=light&token=fg.muted) `fg.muted`           | Use for content that is secondary or that provides additional context but is not critical to understanding the flow of an interface. |
+| ![](https://swatch-sid.vercel.app?mode=light&token=fg.onEmphasis) `fg.onEmphasis` | On emphasis is the text color designed to combine with `-emphasis` backgrounds for optimal contrast.                                 |
+
+| Color roles                                                                     | Usage                                                               |
+| :------------------------------------------------------------------------------ | :------------------------------------------------------------------ |
+| ![](https://swatch-sid.vercel.app?mode=light&token=accent.fg) `fg.accent`       | Use for interactive text or icons like links or buttons.            |
```

Comment:
Note 1: no alt text because we show the name right after the image. the image is decorative

Note 2: we can change the url when we transfer the project to primer's account. tracking issue: https://github.com/primer/swatch/issues/1